### DISCUSSION TITLE
blacksheep/authentication/

### DISCUSSION THREAD

**Discussion by giscus** [Neoteroi/BlackSheep-Docs#22] [Q&A]
# blacksheep/authentication/

The official documentation website of BlackSheep, a fast ASGI web framework for Python asyncio.

https://www.neoteroi.dev/blacksheep/authentication/

**Comment by ctbk** [comment]
Hi! In the "Writing a custom authentication handler" first example I think the header name should be b"Authorization"; also the route `/open/`  is missing at the end of the script:
```
@get('/open/')
async def open(user: Optional[User]):
    if user is None:
        return json({'anonymous': True})
    else:
        return json(user.claims)
```

**Reply by RobertoPrevato** [reply]
Hi! Thank You for the heads up, I fixed the errors. 😄

**Reply by Diman2003** [reply]
I am also facing same issue
